openapi.yaml

```diff
@@ -6050,7 +6691,7 @@ components:
           type: array
           description: >
             A list of tools the model may call. Currently, only functions are supported as a tool.
-            Use this to provide a list of functions the model may generate JSON inputs for.
+            Use this to provide a list of functions the model may generate JSON inputs for. A max of 128 functions are supported.
```

Comment:
This modified description is missing from the TypeSpec. It applies to `CreateChatCompletionRequest` in chat/models.tsp.